prow/config/jobs/sail-operator-release-0.1.yaml

```diff
@@ -0,0 +1,48 @@
+org: istio-ecosystem
+repo: sail-operator
+branches:
+  - release-0.1
+
+support_release_branching: false
+image: gcr.io/istio-testing/build-tools:release-1.22-latest
```

Comment:
Use a real (not floating tag), e.g. https://github.com/istio/test-infra/blob/master/prow/config/jobs/istio-1.22.yaml#L22 - Another job will update the sha when it gets bumped.